<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmpDialog</class>
 <widget class="QDialog" name="EmpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EmpDialog {background-color: rgb(245,255,250)}

QLabel { color: black; }
QLineEdit { border-radius: 3px}

QPushButton
{
	color: black;
	font: 12pt &quot;Segoe UI&quot; bold;
	background-color: rgb(255,255,240);
	border-width: 2px;
    border-radius:3px;
}

QPushButton:hover {background-color: rgb(255,222,173)}

QLabel
{
	font: 12pt &quot;Segoe UI&quot; bold;
}

#equipmentBox
{
	background-color: rgb(255,250,250);
	border-radius: 8px;
}

#membershipBox
{
	background-color: rgb(255,250,250);
	border-radius: 8px;
}
 </string>
  </property>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentDateTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI 13</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quitPushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartPushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="membershipBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>341</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Membership</string>
   </property>
   <widget class="QLabel" name="memberLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Members</string>
    </property>
   </widget>
   <widget class="QLabel" name="numLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="programLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="membershipLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="equipmentBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>341</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Equipment</string>
   </property>
   <widget class="QLabel" name="equipmentLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="treadmillLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>215</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stretchingLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="benchpressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buyBalancePushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellBalancePushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyTreadmillPushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyStretchPushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyBenchpressPushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellTreadmillPushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellStretchPushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellBenchpressPushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
